english/marshmallow.md

```diff
@@ -631,6 +631,8 @@ Take out your [crochet project](crochet/crochet.md) instead and finish that tedd
 
 [Play some DotA 2](play-dota2/play-dota2.md)
 
+[Play Starcraft 2](play-sc2/starcraft2.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.